doc/readme/ABOUT.md

```diff
@@ -4,14 +4,14 @@ Shogun is an open-source machine learning library that offers a wide range of ef
 
 * Supports many languages (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and platforms (Linux/Unix, MacOS and Windows) and integrates with their scientific computing environments.
 
-#### Shogun is state-of-the-art
+#### Shogun is a state-of-art
```

Comment:
this was correct https://en.wikipedia.org/wiki/State_of_the_art :)

Reply:
True, thanks